blueprints/app/files/README.md

```diff
@@ -38,7 +38,7 @@ Make use of the many generators for code, try `ember help generate` for more det
 
 * `<% if (yarn) { %>yarn lint:hbs<% } else { %>npm run lint:hbs<% } %>`
 * `<% if (yarn) { %>yarn lint:js<% } else { %>npm run lint:js<% } %>`
-* `<% if (yarn) { %>yarn lint:js --fix<% } else { %>npm run lint:js -- --fix<% } %>`
+* `<% if (yarn) { %>yarn lint:js:fix<% } else { %>npm run lint:js:fix<% } %>`
```

Comment:
I'd generally prefer to use `lint` and `lint:fix` themselves (instead of `lint:js` + `lint:hbs` + `lint:js:fix` + `lint:hbs:fix`). 

Would you mind updating this section to:

```js
* `<% if (yarn) { %>yarn lint<% } else { %>npm run lint<% } %>`
* `<% if (yarn) { %>yarn lint:fix<% } else { %>npm run lint:fix<% } %>`
```